scripts/GMT_earth_maps.sh

```diff
@@ -8,10 +8,13 @@ gmt set GMT_DATA_SERVER candidate
 let k=0
 I=""
 for set in ${list[@]}; do
-	gmt begin GMT_earth_${set} jpg
+    gmt begin GMT_earth_${set} jpg
         gmt grdimage @earth_${set}_10m -JQ0/7.5c -Rd ${I}
+        if [ set = dist ]; then
+          gmt coast -W0.01p,white
```

Comment:
```suggestion
          gmt coast -W1/0.2p,white -Df
```
1. Need to plot coastlines only, no lakes. 
2. Need to use the full resolution GSHHG data to show the small-area islands, to match the zero-values in the ocean.

Reply:
Updated the images in commit https://github.com/GenericMappingTools/remote-datasets/pull/126/commits/f857778c36b2f8e1ae9fc1d0b4918c8cde414c1d.